Update User Guide for Review Workflows to include RBAC for stages (#1840)
strapi/documentation: docusaurus/docs/user-docs/settings/review-workflows.md

1. Click on the **Add new stage** button.
2. Write the *Stage name*.
3. Select a *Color*.
4. Select *Roles* that can change the stage, if the entity is currently in that review stage.
5. Click on the **Save** button.

By default new stages are appended, but they can be reordered anytime using the ![drag & drop](/img/assets/icons/drag.svg) button.

> :::note

Reviewer: ```suggestion
:::tip
```

> Setting up roles for each stage can be a tedious task. You can either click "Apply to all stages", to apply the current roles to all other stages

Reviewer: ```suggestion
To set up roles for each stage, you can either click "Apply to all stages" to apply the current roles to all other stages of the workflow or use "Duplicate stage" of the stage context menu.
```

We should not say something is easy, difficult, tedious, etc. This remains subjective and the documentation should stay neutral and non-judgemental (esp. towards our own product 😅 — I'd say that if something is "tedious", let's fix it product-side rather than document workarounds 😄).

> of the workflow or use "Duplicate stage" of the stage context menu, to ease the process.

Reviewer 2: Is this a mistake?

Author: Why do you think that?

Reviewer 2: ```suggestion

```

L68 seems to be repeating what's said in L67 🤔 ?

Author: Ah thank - GH must have made a mistake when using the commit suggestion. Will fix!

Author: Fixed and rebased.
